euth_wagtail/templates/email_base.html

```diff
@@ -39,14 +45,43 @@
   <body>
     <table class="content" width="100%" border="0" cellpadding="0" cellspacing="0">
       <tr class="header">
-        <td style="text-align: center; padding-top: 20px">
+        <td style="text-align: center; padding-top: 20px;">
           <img width="50%" src="cid:opin_logo" style="width: 15%;" alt="Logo of the OPIN platform">
         </td>
       </tr>
       <tr>
-        <td style="text-align: center; padding-top:20px;">
-          {% block content %}{% endblock %}
-        </td>
+          <td style="text-align: center; padding-top:20px;">
+              <h2>{% block headline %}{% endblock %}</h2>
+              <p>{% block content %}{% endblock %}</p>
+          </td>
+      </tr>
+      {% block cta %}
+      <tr>
+          <td style="text-align: center; padding-top:20px;">
+              <a href="{% block cta_url %}{% endblock %}" class="button">
+                 {% block cta_label %}{% endblock %}
+              </a>
+          </td>
+      </tr>
+      {% endblock %}
+      <tr style="text-align: center;">
+          <td style="padding-top:20px;">
+              {% block reason %}{% endblock %}
+          </td>
+      </tr>
+      <tr class="footer">
+          <td style="padding-top: 60px;">
+              <hr />
+              {% trans 'OPIN.me is an online platform for youth participation operated by' %}<br />
+              nexus GmbH, Otto-Suhr-Allee 59, D-10585 Berlin<br />
+              Liquid Democracy e.V., Am Sudhaus 2, D-12053 Berlin<br />
+          </td>
+      </tr>
+      <tr>
+          <td class="footer" style="padding-top: 10px; padding-bottom: 10px;">
+              <a href="{{ email.get_host }}/contact">Contact</a><br />
+              <a href="{{ email.get_host }}/terms-of-use">Terms of use</a>
```

Comment:
should we add the links to these pages to SETTINGS and only add them if they are in the SETTINGS